### PR TITLE
Resolve issue 116

### PR DESCRIPTION
**Pull request by japro**
This branch was dedicated to intergrate RASP CPU, memory and compiler into official emuStudio distribution. Now, the most important issues to make it compatible should be done.

### CODE REVIEW

File: plugins/mem/rasp-mem/src/main/java/net/sf/emustudio/rasp/memory/gui/MemoryWindow.java [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        //check if double-click
+        if (evt.getClickCount() == 2) {
+            memoryTable.editCellAt(row, 1);
+        }
+    }//GEN-LAST:event_memoryTableMouseClicked
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String args[]) {
```
Comment (vbmacher) [inline plugins/mem/rasp-mem/src/main/java/net/sf/emustudio/rasp/memory/gui/MemoryWindow.java]: please remove this method

File: README [hunk 2]
```diff
@@ -1 +1,2 @@
 RECOVERY POINT: RASP at this point, instructions modifications are not supported, only operands modification.
+ddddddd
```
Comment (vbmacher) [inline README]: could you please remove this whole commit? It is called `sss` and this is the only added value?

File: plugins/mem/rasp-mem/src/main/java/net/sf/emustudio/rasp/memory/gui/MemoryWindow.java (hunk 1 above)
Comment (japro) [inline plugins/mem/rasp-mem/src/main/java/net/sf/emustudio/rasp/memory/gui/MemoryWindow.java]: It is removed now :)

File: README (hunk 2 above)
Comment (japro) [inline README]: Commit is removed :)

**Pull request merged**